api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/constraint/DateWithinRelativeRange.java

```diff
@@ -56,34 +53,3 @@
      */
     int monthsInFuture();
 }
-
-class DateWithinRelativeRangeValidator implements ConstraintValidator<DateWithinRelativeRange, LocalDate> {
```

Comment:
I quite liked having the constraint and the validator in the same file! Not enough to argue about it though.

Reply:
I had to do this because there is no way for the unit test to construct the validator otherwise, the only way this was working in tests previously was because you were using Mockito which can access the package private class.